.gitignore

```diff
@@ -0,0 +1 @@
+dependencies
```

Comment:
I've seen this in your `.gitignore` files a couple of times, which leads me to believe you have a misunderstanding. There's nothing magical about the word `dependencies` when placed in your `.gitignore` file. Unless you have a directory or file somewhere in your project named `dependencies`, this does nothing.

Your project contains two sets of potential dependencies: one from Bower, another from NPM. Your `bower.json` file enumerates the dependencies that your project relies on in the browser, and the `bower` command installs these pacakges into the `bower_components/` directory. The `npm` command similarly installs packages into the `node_modules/` directory for use on the command line or in Node JS scripts. You might also be tempted to download packages from the internet _outside_ of these mechanisms and use them in your project. These would _also_ constitute dependencies for your project, but they're harder to track.

In avoiding [dependency hell](http://en.m.wikipedia.org/wiki/Dependency_hell), the Internets has given us [package managers](http://en.m.wikipedia.org/wiki/Package_manager) with manifest files like `bower.json` and `package.json` to track the specific version of a library or software package that we intend to use in our projects.